src/main/java/kuznetsov/Bank_Account/controller/AccountController.java

```diff
@@ -0,0 +1,113 @@
+package kuznetsov.Bank_Account.controller;
+
+import jakarta.validation.constraints.NotBlank;
+import kuznetsov.Bank_Account.exception.InvalidPinCodeException;
+import kuznetsov.Bank_Account.exception.NotFoundException;
+import kuznetsov.Bank_Account.exception.OperationBankException;
+import kuznetsov.Bank_Account.model.dto.AccountDto;
+import kuznetsov.Bank_Account.model.dto.OperationDto;
+import kuznetsov.Bank_Account.model.entity.Account;
+import kuznetsov.Bank_Account.serviceImpl.AccountServiceImpl;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/bank")
```

Comment:
Понятно, что логика проекта не для внешних клиентов и вообще он тестовый, но если упустить версионирование, то в будущем его поддерживать проблемно
```suggestion
@RequestMapping("api/v1/bank")
```